tests/functional/test_templates.py

```diff
@@ -69,3 +69,55 @@ def test_templates_for_empty_titles():
                 rel_file = os.path.join(rel_dir, file_name)
                 template = env.get_template(rel_file)
                 assert "title" in template.blocks or "title_base" in template.blocks
+
+
+def test_render_templates():
+    """
+    Test if all HTML templates are rendered without Jinja exceptions.
+    see https://github.com/pypa/warehouse/issues/6634
+    """
+    dir_name = os.path.join(os.path.dirname(warehouse.__file__), "templates")
+
+    env = Environment(
+        loader=FileSystemLoader(dir_name),
+        extensions=[
+            "jinja2.ext.i18n",
+            "warehouse.utils.html.ClientSideIncludeExtension",
+        ],
+        cache_size=0,
+    )
+
+    env.filters.update(
+        {
+            "format_date": "warehouse.i18n.filters:format_date",
+            "format_datetime": "warehouse.i18n.filters:format_datetime",
+            "format_rfc822_datetime": "warehouse.i18n.filters:format_rfc822_datetime",
+            "format_number": "warehouse.i18n.filters:format_number",
+            "format_classifiers": "warehouse.filters:format_classifiers",
+            "classifier_id": "warehouse.filters:classifier_id",
+            "format_tags": "warehouse.filters:format_tags",
+            "json": "warehouse.filters:tojson",
+            "camoify": "warehouse.filters:camoify",
+            "shorten_number": "warehouse.filters:shorten_number",
+            "urlparse": "warehouse.filters:urlparse",
+            "contains_valid_uris": "warehouse.filters:contains_valid_uris",
+            "format_package_type": "warehouse.filters:format_package_type",
+            "parse_version": "warehouse.filters:parse_version",
+            "localize_datetime": "warehouse.filters:localize_datetime",
+        }
+    )
```

Comment:
Perhaps we can automatically generate this list?

Reply:
It makes sense, would you have any suggestions on how to do that?

Reply:
Yeah, might be a bit much for this PR. At the very least, can we not duplicate this between the two tests in this file?